api/document/urls.py

```diff
@@ -1,8 +1,9 @@
 from django.conf.urls import url
 
-from document.views import TreeView
+from document.views import TreeView, editor
 
 urlpatterns = [
+    url(r'^editor$', editor),
```

Comment:
Not for now, but something to think about: We may want to put this at `[domain]/admin/editor` to keep everything restricted in the `admin` path. In particular, I'm thinking about our CORS configuration.

We might also consider moving the admin to a separate domain.